runtime/runc_jailer.go

```diff
@@ -138,10 +138,18 @@ func newRuncJailer(
 
 func (j *runcJailer) prepareBindMounts(mounts []*proto.FirecrackerDriveMount) error {
 	for _, m := range mounts {
-		err := j.bindMountFileToJail(m.HostPath, filepath.Join(j.RootPath(), m.HostPath))
-		if err != nil {
+		stat := syscall.Stat_t{}
+		if err := syscall.Stat(m.HostPath, &stat); err != nil {
 			return err
 		}
+		// Only bindMount regular file.
```

Comment:
we need to be absolutely be sure if this assumption won't break anything.

Reply:
I am pretty sure it won't.